### PR TITLE
Update Spanish

### PR DESCRIPTION
**Pull request by RockLee-BC**
I believe this is everything, anything will tell me :)

### CODE REVIEW

File: Themes/default/languages/LightPortal/.spanish_es.php [hunk 1]
```diff
@@ -238,11 +238,43 @@
...
+$txt['lp_plugin_components_author'] = 'Autor del componente';
+
+$txt['lp_plugin_option_name'] = 'Nombre de la opción (latín)';
+$txt['lp_plugin_option_type'] = 'Tipo de opción';
+$txt['lp_plugin_option_type_set'] = array('Campo de texto', 'Dirección web', 'Color de entrada', 'Número de entrada', 'Casilla de verificación', 'Selección múltiple ', 'Seleccione');
+$txt['lp_plugin_option_default_value'] = 'Valor por defecto';
+$txt['lp_plugin_option_variants'] = 'Valores posibles';
+$txt['lp_plugin_option_variants_placeholder'] = 'Varias opciones separadas por comas';
+$txt['lp_plugin_option_translations'] = 'Localización';
+$txt['lp_plugin_new_option'] = 'Agregar opción';
```
Comment (dragomano) [inline Themes/default/languages/LightPortal/.spanish_es.php:278]: Compare with `.english.php` file in the **develop** branch

**Review by dragomano — CHANGES_REQUESTED** [review]

File: Sources/LightPortal/addons/Likely/langs/spanish_es.php [hunk 2]
```diff
@@ -16,7 +25,7 @@
 		'telegram' => 'Enviar',
 		'linkedin' => 'Compartir',
 		'whatsapp' => 'Enviar',
-		'viber' => 'Enviar',
+		'viber' => 'Send',
```
[the remark this answers is not in the document]
Comment (RockLee-BC) [reply Sources/LightPortal/addons/Likely/langs/spanish_es.php]: Ops I overlooked it, my bad

File: Sources/LightPortal/addons/BoardIndex/langs/spanish_latin.php [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<?php
+// Spanish translation by Rock Lee https://www.bombercode.net | Rev. 1.0
+
+return [
+	'description' => 'Configuraciones adicionales de <a class="bbc_link" href="%1$s">la página de inicio del foro</a> si el portal está activado',
+  'allow_for_spiders' => 'Permitir la indexación de la página de inicio del foro',
```
Comment (dragomano) [inline Sources/LightPortal/addons/BoardIndex/langs/spanish_latin.php:6]: There is still not enough tabs

**Review by dragomano — APPROVED** [review]
Good job, gracias :)

**Comment by RockLee-BC** [comment]
no problem :D I see if I look at the documentation and translate, gran trabajo 👍

**Comment by dragomano** [comment]
In any case, use develop branch of this repository to fetch the latest changes and to push your changes.

**Pull request merged**